make/common/mariadb/Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM vmware/photon:1.0
 
 RUN tdnf distro-sync -y || echo \
     && tdnf install -y sed shadow procps-ng gawk gzip sudo net-tools \
-    && groupadd -r -g 999 mysql && useradd --no-log-init -r -g 999 -u 999 mysql \
+    && groupadd -r -g 10000 mysql && useradd --no-log-init -r -g 10000 -u 10000 mysql \
```

Comment:
do we need to run some chown 10000:10000 -R /var/lib/mysql  in the docker-entrypoint.sh file for the db?

Reply:
entrypoint.sh of mariadb will call `chown`

https://github.com/vmware/harbor/blob/master/make/common/mariadb/docker-entrypoint.sh#L70